src/instruments/src/Common/gauges.tsx

```diff
@@ -171,3 +173,124 @@ export const VerticalSegment: FC<VerticalSegmentProps> = ({ x, y, height, rangeS
+    // const start = startAngle > 180 ? 360 - startAngle : startAngle;
+    let angleInRadians = startAngle > endAngle
+        ? startAngle + (valuePercentage * (360 - startAngle + endAngle)) - 90
+        : startAngle + (valuePercentage * (endAngle - startAngle)) - 90;
+    // let angleInRadians = angleInRadiansOne > 359 ? 360 - angleInRadiansOne : angleInRadiansOne;
+    console.log(`Angle is ${angleInRadians}`);
+    angleInRadians *= (Math.PI / 180.0);
+    console.log(`Angle is radians is ${angleInRadians}`);
+    return ({
+        x: Math.cos(angleInRadians),
+        y: Math.sin(angleInRadians),
+    });
+};
+
+type GaugeMarkerComponentType = {
+    value: number,
+    x: number,
+    y: number,
+    min: number,
+    max: number,
+    radius: number,
+    startAngle: number,
+    endAngle: number,
+    className: string,
+    showValue?: boolean,
+    indicator?: boolean,
+    outer?: boolean,
+    multiplier?: number,
+};
+
+const GaugeMarkerComponentNoMemo = ({ value, x, y, min, max, radius, startAngle, endAngle, className, showValue, indicator, outer, multiplier } : GaugeMarkerComponentType) => {
+    const dir = valueRadianAngleConverter(value, min, max, endAngle, startAngle);
+
+    if (typeof multiplier === 'undefined') multiplier = 1.15;
+
+    let start = {
+        x: x + (dir.x * radius * 0.9),
+        y: y + (dir.y * radius * 0.9),
+    };
+    let end = {
+        x: x + (dir.x * radius),
+        y: y + (dir.y * radius),
+    };
+
+    if (outer) {
+        start = {
+            x: x + (dir.x * radius),
+            y: y + (dir.y * radius),
+        };
+        end = {
+            x: x + (dir.x * radius * multiplier),
+            y: y + (dir.y * radius * multiplier),
+        };
+    }
+
+    if (indicator) {
+        start = { x, y };
+
+        end = {
+            x: x + (dir.x * radius * multiplier),
+            y: y + (dir.y * radius * multiplier),
+        };
+    }
+
+    // Text
+    const multiplierX = value >= 8 ? 0.8 : 0.7;
+    const multiplierY = value >= 8 ? 0.52 : 0.7;
+    const pos = {
+        x: x + (dir.x * (radius * multiplierX)),
+        y: y + (dir.y * (radius * multiplierY)),
+    };
+
+    const textValue = !showValue ? '' : Math.abs(value).toString();
+
+    return (
+        <>
+            <line x1={start.x} y1={start.y} x2={end.x} y2={end.y} className={className} />
+            <text x={pos.x} y={pos.y} className={className} alignmentBaseline="central" textAnchor="middle">{textValue}</text>
+        </>
+    );
+};
+export const GaugeMarkerComponent = React.memo(GaugeMarkerComponentNoMemo);
+
+interface GaugeComponentProps {
+    x: number,
+    y: number,
+    radius: number,
+    startAngle: number,
+    endAngle: number,
+    className: string,
+    manMode: boolean,
+    children?: React.ReactNode,
+}
+
+export const GaugeComponentNoMemo: React.FunctionComponent<GaugeComponentProps> = ({ x, y, radius, startAngle, endAngle, className, children, manMode }) => {
```

Comment:
**suggestion**: use `FC` instead of `React.FunctionComponent`

```suggestion
export const GaugeComponentNoMemo: FC<GaugeComponentProps> = ({ x, y, radius, startAngle, endAngle, className, children, manMode }) => {
```